docs/tempo/website/configuration/manifest.md

```diff
@@ -273,14 +280,14 @@ ingester:
     unregister_on_shutdown: true
     address: 127.0.0.1
     port: 0
-    id: Annanays-Mac.local
+    id: hostname
   concurrent_flushes: 16
   flush_check_period: 30s
   flush_op_timeout: 5m0s
   trace_idle_period: 30s
   max_block_duration: 1h0m0s
   max_block_bytes: 1073741824
-  complete_block_timeout: 6m0s
```

Comment:
I'm curious why some of these changed, have we not updated this file for a while?

Reply:
yes, a lot of these changes are just b/c we haven't updated in a bit. that one in particular was my miss in the tenant index PR.

honestly this file should be auto generated in CI :P

Reply:
https://github.com/grafana/tempo/issues/890